common/lib/common-definitions/package.json

```diff
@@ -60,7 +60,23 @@
     "typescript-formatter": "7.1.0"
   },
   "typeValidation": {
-    "version": "0.21.1000",
-    "broken": {}
+    "version": "1.0.0",
+    "broken": {
+      "InterfaceDeclaration_ITelemetryBaseEvent": {
+        "backCompat": false
+      },
+      "InterfaceDeclaration_ITelemetryErrorEvent": {
+        "backCompat": false
+      },
+      "InterfaceDeclaration_ITelemetryGenericEvent": {
+        "backCompat": false
+      },
+      "InterfaceDeclaration_ITelemetryProperties": {
+        "backCompat": false
+      },
+      "InterfaceDeclaration_ITelemetryPerformanceEvent": {
+        "backCompat": false
+      }
```

Comment:
@markfields Can you take a look at this? The typetests try to compare the current version on the branch to the last released major of the package. For common-defs, that's 0.20.0. So these changes represent "breaks" in the interfaces listed. This may be expected - but I want to verify these are expected API changes.